.vscode/launch.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
@henrik-me : why do we need to commit this file?

Reply:
It is stuff auto generated for those who work out of VSCode. I can put it in gitignore if you think that is the right thing?

Reply:
personally, I would ignore the .vscode and .vs folders (that's the guidance I've been giving for samples). Now if there are good reasons to keep them, you know better :)